### DISCUSSION TITLE
We migrated from GitBook and are super happy with Retype!

### DISCUSSION THREAD

**Discussion by etienne-bechara** [retypeapp/retype#171] [Show and tell]
First of all we would like to congratulated everybody involved in this awesome project!

Our company offers an API to query data regarding Brazilian companies, and we used to host our public documentation in GitBook.

Recently, we got annoyed with their paywall to access some features, and this fact paired with their poor page loading performance led us to look for an alternative.

We found Retype via Google and decided to give it a try, we ended up being so happy with the results that we fully migrated our documentation (~30 pages) in a couple of days.

Here are the results, unfortunately it is all in Portuguese, but I think it gives an general idea of the usage:
https://docs.cnpja.com

**Why we moved**:

- Loading performance: Can't even compare, Retype is much faster.
- Free: We ended up having to pay for a lot of collaboration features (which is not really relevant to us) just to have access to simple branding customizations.
- Design: Both look great, but Retype really impresses considering it's not paid. We could see a lot of care was put in it. Also, dark theme!

**Things we miss**:

- API component: They have an interesting component that allows API design to look neat by organizing the URL, path, headers, etc. We kind of invented an alternative using badges and tables that is enough for explaining, but the previous design certainly looked better. [Example](https://docs.cnpja.com/api/data/office/#consulta-cnpj).
- Content search: We understand the complexity of this one, but was a nice feature since we also publish things like terms of usage and privacy policy within the docs.
- Custom scrollbars: Most of our visitors are on Windows, and since we keep the left menu expanded it, as well as have a lot of code components, it usually leads to the ugly scrollbars being displayed.
- Docs variants: Currently not in use, but we have some plans for the future, they have a feature where you could create a selectable alternative of the website intended for versioning or localization.

Most of the content above is already being discussed in other topics, but we just wanted to give an overview of the pros and cons we found during our comparison.

**Comment by DannyBen** [comment]
Hey - congratulations on the move, your site looks great.
As a former GitBook user myself, I can definitely relate.

**Comment by LoneDev6** [comment]
The only reason why I'm not moving is the 100 pages limit

**Reply by fabriciomurta** [reply]
Check out discussion #92 you just commented, there Geoffrey shares something that might interest you and help you get started at once.

**Reply by LoneDev6** [reply]
Thanks, but a temporary unlock key is not the solution

**Comment by geoffreymcgill** [comment]
Hi @etienne-bechara,

With the upcoming Retype v3.0 release, you will be able to set `locale: pt` or `locale: pt-BR` in your `retype.yml` project config file and Retype will build all the system interface with Portuguese.

More details available in the [translation](https://github.com/retypeapp/retype-translations) repo and related [docs](https://retypeapp.github.io/retype-translations/).

If you have a chance, we would appreciate your feedback on the [`pt`](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.pt.resx) and [`pt-BR`](https://github.com/retypeapp/retype-translations/blob/main/src/UserInterface.pt-FR.resx) translations.

Hope this helps.
